galaxy.yml

```diff
@@ -0,0 +1,20 @@
+namespace: community
+name: FIXME
+version: 0.1.0
+readme: README.md
+authors:
+  - YOUR NAME (github.com/YOURGITHUB)
+description: null
+license_file: COPYING
+tags:
+  - acme
```

Comment:
I would remove these example tags ;-)

Maybe add a dummy tag, and a link to the galaxy.yml reference which says what kind of tags are allowed? (all lower-case, no space, no dash, ...)